needy/projects/cmake.py

```diff
@@ -17,7 +17,11 @@ def configuration_keys():
 
     @staticmethod
     def is_valid_project(definition, needy):
-        return definition.target.platform.is_host() and os.path.isfile('CMakeLists.txt')
+        if not os.path.isfile('CMakeLists.txt'):
+            return False, 'no CMakeLists.txt found in project root'
+        if not definition.target.platform.is_host():
+            return False, 'At this time, cross compilation of CMake projects is not supported'
```

Comment:
Let's try to be consistent with capitalization here. Drop it to lowercase like your other reasons. I would re-order the wording to be a little more to-the-point: "cross-compilation for cmake not yet supported"